scripts/build/Platform/Windows/build_config.json

```diff
@@ -1,4 +1,46 @@
 {
+  "public_metrics_pipe": {
```

Comment:
I'm a bit confused on how this is defining when these metrics are collected. Is this running its own forked periodic pipeline? Shouldn't metrics get uploaded as part of existing pipelines?

Reply:
The `periodic-incremental-daily-internal` tag will mean that this will run in any internal Jenkins that is not the O3DE one. I did question whether this is necessary or just a sandboxing measure while this is in active development

Reply:
These metrics are intended to be ran only on the internal Jenkins because these metrics are not yet public.

They are running in a separate "pipe" in the current Periodic Incremental Daily Jenkins job because all current build steps are "blocking" which means if a build/test step fails, it will not trigger the metrics step.